make/langtools/netbeans/nb-javac/pom-nb-javac.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+  http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>eu.doppel-helix.com.github.oracle.nb-javac</groupId>
+  <artifactId>nb-javac</artifactId>
+  <version>TEMPLATE</version>
+  <packaging>jar</packaging>
+
+  <name>nb-javac</name>
+  <description>"nb-javac" is a patched version of OpenJDK "javac", i.e., the Java compiler. This has long been part of NetBeans, providing a highly tuned Java compiler specifically for the Java editor i.e., parsing and lexing for features such as syntax coloring, code completion.</description>
+  <url>https://github.com/oracle/nb-javac</url>
+
+  <licenses>
+      <license>
+          <name>GPLv2-CPE</name>
+          <url>https://openjdk.java.net/legal/gplv2+ce.html</url>
+          <distribution>repo</distribution>
+      </license>
+  </licenses>
+
+  <scm>
+    <connection>scm:git:https://github.com/oracle/nb-javac.git</connection>
+    <developerConnection>scm:git:git@github.com:oracle/nb-javac.git</developerConnection>
+    <url>https://github.com/oracle/nb-javac</url>
+  </scm>
+
+  <developers>
+    <developer>
+      <email>mblaesing@doppel-helix.eu</email>
+      <name>Matthias Bläsing</name>
+      <url>https://github.com/matthiasblaesing/</url>
+      <roles>
+        <role>Developer</role>
+      </roles>
```

Comment:
Congratulation on becoming `nb-javac` developer!

I assume your entry should stay as you have contributed the maven deployment script. In addition to that contacts to @Akshay-Gupta-Oracle, @singh-akhilesh or some generic contact to maintainers of this repository shall be added as well.

Reply:
No I should not. I won't publish nb-javac, but being listed in the POM is a requirement to be able to publish to maven central. I tested publishing to the snapshot repository and the staging repository and for that to work, I had to be listed. Just as the groupId is currently not usable for the oracle team, this needs to be updated accordingly. 

That is also the reason, that I don't intent to make the groupId a variable.  The pom-nb-javac.xml is a configuration file per group that redistributes. The only thing that regularly changes over time is the version, the developers will be pretty constant and the same is true for the group id.

Reply:
According to the first paragraph the `nb-javac` maintainers are supposed to __replace__ the developer section with their own information. OK, let them do it.